agagd/agagd_core/tables.py

```diff
@@ -237,6 +237,10 @@ class TournamentPlayedTable(tables.Table):
     date = tables.Column(default="Unknown")
     won = tables.Column(verbose_name="Won", default=0)
     lost = tables.Column(verbose_name="Lost", default=0) 
+    rating = tables.LinkColumn(
+            'member_detail',
+            kwargs={'member_id':tables.A('ratings.elab_date')})
+
```

Comment:
This should be a regular column. I am not sure why we would want a `LinkColumn` on the rating as we already have a `LinkColumn` for the tournament.

Locally, I am not getting any rating for this column.
![image](https://user-images.githubusercontent.com/5054653/103465122-c5894580-4ced-11eb-8dc9-096b6e56bede.png)

Reply:
I was wondering where that went. I left that in my repo for a placeholder I was working on. Ill back that out. Its supposed to show the rating for that tournament for that player inline. See issue #3 for more info.

Reply:
That sounds good. We should have a separate pull for #3. I think that complicates your changes here.

Reply:
Pulled that out of the code. #3 will be a separate thing.